<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>312</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Smart Parking System - Đăng Nhập</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: #f5f5f5;
}</string>
  </property>
  <layout class="QVBoxLayout" name="main_layout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="content_frame">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame {
    background-color: white;
    border: none;
}</string>
     </property>
     <layout class="QVBoxLayout" name="content_layout">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>40</number>
      </property>
      <property name="topMargin">
       <number>30</number>
      </property>
      <property name="rightMargin">
       <number>40</number>
      </property>
      <property name="bottomMargin">
       <number>30</number>
      </property>
      <item>
       <widget class="QLineEdit" name="username_input">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    padding: 10px;
    border: 1px solid #d0d0d0;
    border-radius: 5px;
    background-color: #fafafa;
    font-size: 11px;
}
QLineEdit:focus {
    border: 2px solid #0078d4;
    background-color: white;
}</string>
        </property>
        <property name="placeholderText">
         <string>Nhập tên đăng nhập...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="password_input">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
    padding: 10px;
    border: 1px solid #d0d0d0;
    border-radius: 5px;
    background-color: #fafafa;
    font-size: 11px;
}
QLineEdit:focus {
    border: 2px solid #0078d4;
    background-color: white;
}</string>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
        <property name="placeholderText">
         <string>Nhập mật khẩu...</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="show_password_checkbox">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Hiển thị mật khẩu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="remember_checkbox">
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Ghi nhớ tên đăng nhập</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="button_layout">
        <property name="spacing">
         <number>15</number>
        </property>
        <item>
         <widget class="QPushButton" name="login_button">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>45</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #0078d4;
    color: white;
    border: none;
    border-radius: 5px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: #0063b1;
}
QPushButton:pressed {
    background-color: #004a94;
}
QPushButton:disabled {
    background-color: #cccccc;
    color: #666;
}</string>
          </property>
          <property name="text">
           <string>✓ Đăng Nhập</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exit_button">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>45</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: #f3f3f3;
    color: #333;
    border: 1px solid #cccccc;
    border-radius: 5px;
}
QPushButton:hover {
    background-color: #e8e8e8;
}
QPushButton:pressed {
    background-color: #dcdcdc;
}</string>
          </property>
          <property name="text">
           <string>✕ Thoát</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
